package.json

```diff
@@ -75,7 +75,6 @@
     "devtools-splitter": "^0.0.6",
     "devtools-utils": "^0.0.12",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint-plugin-file-header": "0.0.1",
     "eslint-plugin-import": "^2.8.0",
```

Comment:
not part of this work but maybe we move this to devDependencies as well .. thanks